### PR TITLE
PMM-6729 RN 2.11.0

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

File: docs/release-notes/2.11.0.md [hunk 1]
```diff
@@ -24,25 +24,30 @@ performance.
...
 
 
 
 ## Improvements
 
-* [PMM-6727](https://jira.percona.com/browse/PMM-6727): Update Grafana plugin grafana-polystat-panel to latest v1.2.2
+* [PMM-6727](https://jira.percona.com/browse/PMM-6727): Grafana plugin updates: grafana-polystat-panel=1.2.2, grafana-piechart-panel=1.6.1
 * [PMM-6625](https://jira.percona.com/browse/PMM-6625): Default sort to "Average - descending" on all dashboards
 * [PMM-6609](https://jira.percona.com/browse/PMM-6609): MySQL Instances Compare & Summary dashboards: Changed metric in 'MySQL Internal Memory Overview'
+* [PMM-6598](https://jira.percona.com/browse/PMM-6598): Improved wording on Dashboard image 'Share Panel'
```
Comment (rnovikovP) [inline docs/release-notes/2.11.0.md:37]: Improved wording and link to instructions on how to configure image sharing added

File: docs/release-notes/2.11.0.md [hunk 2]
```diff
@@ -24,25 +24,30 @@ performance.
 
 ## New Features
 
-* [PMM-6567](https://jira.percona.com/browse/PMM-6567): Management API for new pg_stat_monitor PostgreSQL extension
+* [PMM-6567](https://jira.percona.com/browse/PMM-6567): New PostgreSQL extension 'pg_stat_monitor'
```
Comment (rnovikovP) [inline docs/release-notes/2.11.0.md:27]: Technical Preview of new PG extension  .... . We need to make it clear  - it's not GA  yet

File: docs/release-notes/2.11.0.md (hunk 1 above)
Comment (PaulJacobs-percona) [reply docs/release-notes/2.11.0.md:37]: Edited Jira summary:
"Dashboard image sharing (Share Panel): Improved wording with link to configuration instructions"

File: docs/release-notes/2.11.0.md (hunk 2 above)
Comment (PaulJacobs-percona) [reply docs/release-notes/2.11.0.md:27]: Edited in Jira summary:
"Technical preview of new PostgreSQL extension 'pg_stat_monitor'"

**Pull request merged**